test/browser/step_definitions/F2FBackendValidation.step.js

```diff
@@ -65,12 +68,25 @@ Then(
     const decodedBody = JSON.parse(
       Buffer.from(rawBody.replace(/\W/g, ""), "base64url").toString()
     );
-    // Strength Score
-    expect(decodedBody.vc.evidence[0].strengthScore).to.equal(3);
-    // Validity Score
-    expect(decodedBody.vc.evidence[0].validityScore).to.equal(2);
-    // Verification Score
-    expect(decodedBody.vc.evidence[0].verificationScore).to.equal(3);
+    const yotiMockIdId = this.yotiSessionId.substr(
+      this.yotiSessionId.length - 4
+    );
+    if (vcResponseData) {
+      // Strength Score
+      testHarness.checkVerifiableCredentialValue(decodedBody, yotiMockIdId, eval(
+        "vcResponseData.s" + yotiMockIdId + ".strengthScore"
+      ), "strengthScore")
+      // // Validity Score
+      testHarness.checkVerifiableCredentialValue(decodedBody, yotiMockIdId, eval(
+        "vcResponseData.s" + yotiMockIdId + ".validityScore"
+      ), "validityScore")
+      // // Verification Score
```

Comment:
[nit] we seem to have double //
```suggestion
      // Verification Score
```